tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py

```diff
@@ -29,13 +29,45 @@ class ActionDetection(Adapter):
     def parameters(cls):
         parameters = super().parameters()
         parameters.update({
-            'priorbox_out': StringField(description="Name of layer containing prior boxes in SSD format."),
             'loc_out': StringField(description="Name of layer containing box coordinates in SSD format."),
             'main_conf_out': StringField(description="Name of layer containing detection confidences."),
+            'new_net': BoolField(
```

Comment:
could you please provide a more descriptive name for this parameter? Yes, it is new architecture for this release, but what about future, when we will need to support other topology changes?